ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -67,3 +67,12 @@ spec:
         - mirrors:
           - registry.example.com:5000
           source: registry.redhat.io
+    #
+    # These CRs are to enable crun on master and worker nodes for 4.13+ only
+    #
+    # It's strongly recommended to include these CRs as day-0 extra manifests to
+    # avoid an extra reboot.
+    - fileName: optional-extra-manifest/enable-crun-master.yaml
+      policyName: "config-policy"
+    - fileName: optional-extra-manifest/enable-crun-worker.yaml
+      policyName: "config-policy"
```

Comment:
From a logical standpoint the common configuration is the right place for this. However, this will cause a reboot due to the runtime change. If that reboot occurs while day-2 operators are installing it will create a race condition that can cause those installs to fail. For that reason I think we need to move this to the group config so that it will occur after the operator installation.
If the user follows our advice and puts this in the day0 machine config they won't run into this race at installation. However if they choose not to do it day0, or if these get applied as part of an upgrade where operators are upgraded (and the user keeps our initial "wave" ordering) they will get hit by this. Please add a note to the group indicating why it is there.

Reply:
Very good point! Will move to group.